frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -561,8 +563,8 @@ function getSearchableTokenFieldPlaceholder(
     ),
   );
 
-  if (names.size > 1) {
-    placeholder = t`Search`;
+  if (names.size !== 1 || getSourceType(parameter) != null) {
+    placeholder = t`Search the list`;
```

Comment:
This matches the placeholder of the `list` mode

Reply:
The thing here is that we cannot use names of connected fields for placeholders, otherwise we would get `Search by <connected field name>`, but in reality we are searching by some field from the custom source card